### DISCUSSION TITLE
Help Request - Fixing lua Require

### DISCUSSION THREAD

**Discussion by Starnt-xyz** [neovim/neovim#26727] [Q&A] [answered]
SO everything works when i run nvim from .config/nvim
But when i run it from anywhere else it fails
i use require like this
require "plugins.config.custom"
and anywhere else it cant find them

**Comment by zeertzjq** [comment, marked as answer]
You need to put them under `.config/nvim/lua/` instead.

**Reply by Starnt-xyz** [reply]
ok, then do i require them as "lua.plugins" etc or do i just leave them like this?

**Reply by zeertzjq** [reply]
No, leave them as `plugins.`.
